code/modules/cargo/exports/materials.dm

```diff
@@ -84,3 +84,15 @@
 	material_id = /datum/material/glass
 	export_types = list(/obj/item/stack/sheet/glass, /obj/item/stack/ore,
 		/obj/item/shard)
+
+/datum/export/material/hot_ice
+	cost = 400
+	message = "cm3 of Hot Ice"
+	material_id = /datum/material/hot_ice
+	export_types = /obj/item/stack/sheet/hot_ice
```

Comment:
Export types for both of these should be lists, metal hydrogen probsbly doesn't need an export_types list either